apps/aecore/lib/aecore/miner/worker.ex

```diff
@@ -25,6 +25,7 @@ defmodule Aecore.Miner.Worker do
 
   @mersenne_prime 2147483647
   @coinbase_transaction_value 100
+  @new_candidate_nonce_count 500
```

Comment:
I would put much smaller number here (~10). AFAIK one cuckoo execution is few seconds.

Reply:
(We might even be able for now to allow ourselves to regenerate every execution without much overhead)

Reply:
in my testing the 500 was reached really quick ~10sec-20sec in our current configuration, lets keep it at 500 now, can change this later